tests/libs/six.py

```diff
@@ -0,0 +1,854 @@
+"""Utilities for writing code that runs on Python 2 and 3"""
```

Comment:
we need to include the entire six library ourselves?

Reply:
I'm not sure.  If I try to install it with pip3 I get the following error.

```
bad magic number in 'libs.six': b'\x03\xf3\r\n'
```

Reply:
okay.   good enough for now - but we shouldn't need to have python2 compatibility soon (if not already).

Reply:
Now that I understand what six.py does and that we are only running this in Python 3, I was able to modify the libraries to remove any six.py dependencies.